tests/rhtap-demo/rhtap-demo.go

```diff
@@ -112,7 +112,7 @@ var _ = framework.RhtapDemoSuiteDescribe(Label("rhtap-demo"), func() {
 					Expect(err).NotTo(HaveOccurred())
 
 					if !CurrentSpecReport().Failed() {
-						if err := fw.AsKubeAdmin.HasController.DeleteAllComponentsInASpecificNamespace(namespace, 60*time.Second); err != nil {
+						if err := fw.AsKubeAdmin.HasController.DeleteAllComponentsInASpecificNamespace(namespace, 90*time.Second); err != nil {
```

Comment:
`DeleteAllComponentsInASpecificNamespace` is also used in [environments.go](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/remotesecret/environments.go#L62). Should we increase there too?